### DISCUSSION TITLE
Add days to the last modified date

### DISCUSSION THREAD

**Discussion by ellanrk** [microsoft-search/pnp-modern-search#3598] [Q&A]
Is there any expression in the current solution that will help me to add 'x' amount of days to the 'last modified date'?

**Comment by wobba** [comment]
Did you read the documentation at https://microsoft-search.github.io/pnp-modern-search/usage/search-results/tokens/#date-tokens? Or maybe https://github.com/helpers/handlebars-helpers?tab=readme-ov-file#moment if you need to change what is displayed. Also getDate https://microsoft-search.github.io/pnp-modern-search/extensibility/templating/#handlebars-helpers.
